crates/scroll/hardforks/src/hardfork.rs

```diff
@@ -0,0 +1,293 @@
+//! Hard forks of scroll protocol.
+
+use alloc::{boxed::Box, format, string::String, vec};
+use core::{
+    any::Any,
+    fmt::{self, Display, Formatter},
+    str::FromStr,
+};
+
+use alloy_chains::{Chain, NamedChain};
+use reth_ethereum_forks::{hardfork, ChainHardforks, EthereumHardfork, ForkCondition, Hardfork};
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+hardfork!(
+    /// The name of the Scroll hardfork
+    ///
+    /// When building a list of hardforks for a chain, it's still expected to mix with
+    /// [`EthereumHardfork`].
+    ScrollHardfork {
+        /// Bernoulli: <https://scroll.io/blog/blobs-are-here-scrolls-bernoulli-upgrade>.
+        Bernoulli,
```

Comment:
So Archimedes will be enabled by default, right?

Reply:
fixed